### DISCUSSION TITLE
Payload Size Limit

### DISCUSSION THREAD

**Discussion by captain-http** [quirrel-dev/quirrel#970] [Q&A] [answered]
Does Quirrel have limits for the payload size of a job?

**Comment by Skn0tt** [comment, marked as answer]
Hey @babasbot! No hard limits enforced on the Quirrel side, but Redis limits value size to 512mb. So that's probably a hard limit :) Other than that, you'll probably find that higher payload sizes mean higher networking latency + higher memory usage, which definitely can cause a problem if you have a lot of these huge jobs.

**Reply by captain-http** [reply]
Thanks for the answer @Skn0tt :)
